_Week8/Group3/README.md

```diff
@@ -0,0 +1,74 @@
+# WEEK 8 HW - Group 3
+
+## Motivation
+
+For this week's simulation assignment, we chose to illustrate the importance of intracluster correlation and the resulting design effect in clustered research designs.
+
+### ICC
+
+The intracluster correlation coefficient is a measure of how data point within clusters are related. The ICC is an important consideration when conducting power calculations and interpreting the results of data from a clustered study design. This is because similarity among the observations in clusters makes it more difficult to detect the true differences between study arms by reducing response variability. The ICC measures this similarity by comparing within cluster variance to between cluster variance.
+
+Note that accounting for similarities among clustered subjects usually results in power loss. Thus, with higher levels of intracluster correlation, more observations are required to obtain the same minimal detectable effect.
+
+A high icc ($\rho$) represents comparatively high "between clusters" variance.
+
+If $\rho=1$, then all responses within a cluster are identical.
+
+If $\rho=0$, then there is no correlation of responses within a cluster and the observations are functionally independent.
```

Comment:
Note $\rho$ can be _negative_! Consider a case where, for example, treatment assignment is done among _pairs of spouses_ – then the ICC is -1. That is, if you know one spouse's assignment you know the other's is the _opposite_!

Reply:
Unfortunately, with the way we set up our simulation, I get an error in Stata when trying to set a negative $/rho$. Apparently, the dependent variable 'score' is collinear with the intercept term. Perhaps I am not sufficiently familiar with the ICC, but I also don't know how ICC would be negative--my current understanding of it is that ICC is a fraction of variances (which are negative negative), but this may be a simplification of the formula.